doc/cephadm/services/osd.rst

```diff
@@ -138,6 +138,12 @@ There are a few ways to create new OSDs:
 
     ceph orch daemon add osd host1:/dev/sdb
 
+  Advanced OSD creation from specific devices on a specific host:
```

Comment:
Not something for this PR, but when this ends up being backported can you add in which version it will land in (e.g. 16.2.8 for pacific backport if it gets in in time)